### PR TITLE
Local Files Watcher / events handler

### PR DESCRIPTION
**Pull request by tem-tem**
TODO: 

- [x] Full disk access handler
- [x] local directory watcher
- [x] local files requests handler

### CODE REVIEW

File: release/app/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bedrock",
-  "version": "1.3.1",
+  "version": "1.5",
```
Comment (Tammerabi) [inline release/app/package.json]: Should be 1.5.0 (consistent number of digits)

**Review by Tammerabi — APPROVED** [review]
Just version # then good to go

File: release/app/package.json (hunk 1 above)
Comment (omieliekh) [reply release/app/package.json]: It should be 1.4.0, unless there is a reason to increment two versions at once

Comment (Tammerabi) [reply release/app/package.json]: Nope, you're right. Thought I saw 1.4.0 in the wild somewhere

**Review by Tammerabi — APPROVED** [review]

**Pull request merged**